titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 1 to 2_
```python
import seaborn as sns
import matplotlib.pyplot as plt
```
multi line comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/25)._